src/native/eventpipe/ds-ipc-pal-socket.c

```diff
@@ -323,11 +323,8 @@ ipc_socket_create_uds (DiagnosticsIpc *ipc)
 	DS_ENTER_BLOCKING_PAL_SECTION;
 	new_socket = socket (ipc->server_address_family, socket_type, 0);
 #ifndef SOCK_CLOEXEC
-	if (new_socket != DS_IPC_INVALID_SOCKET) {
-		if (fcntl (new_socket, F_SETFD, FD_CLOEXEC) == -1) {
-			EP_ASSERT (!"Failed to set CLOEXEC");
-		}
-	}
+	if (new_socket != DS_IPC_INVALID_SOCKET)
```

Comment:
Should this also be ifdef'd against HOST_WIN32?

Reply:
Its already under #if defined(DS_IPC_PAL_AF_UNIX).